tests/test_add.py

```diff
@@ -200,33 +200,50 @@ def test_command(db, tables_only, tables, views, caplog):
         assert len(rows) == 65235
         assert rows[0] == (1, 'release', 'awards', 100, 301, 100)
 
-        # Check length of field_list field for lowest ids in each summary table.
-        each_table = '(SELECT array_length(field_list, 1) FROM view_data_collection_1.{} order by {} limit 1)'
+        # Check count of keys in field_list field for lowest ids in each summary table.
+        each_table = '''
+            (SELECT
+                count(*)
+            FROM
+                (SELECT
+                    jsonb_each(field_list)
+                FROM (
+                    SELECT
+                       field_list
+                    FROM
+                       view_data_collection_1.{}
+                    ORDER BY
+                       {}
+                    LIMIT 1) AS field_list
+                ) AS each
+            )
```

Comment:
I think the surrounding parentheses are optional (e.g. `SELECT 1 UNION SELECT 2;` works):
```suggestion
            SELECT
                count(*)
            FROM
                (SELECT
                    jsonb_each(field_list)
                FROM (
                    SELECT
                       field_list
                    FROM
                       view_data_collection_1.{}
                    ORDER BY
                       {}
                    LIMIT 1) AS field_list
                ) AS each
```